Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM wisvch/debian:stretch
+MAINTAINER Mark Janssen <mark@praseodym.net>
+
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy && rm -rf /var/lib/apt/lists/*
+COPY florriebot.tar.gz /tmp
```

Comment:
That's for sure a good idea :blush: 

Would it be complicated to do an `npm install` and then clean up afterwards, so that the image reduces in size again?